lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,11 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://scientificrussia.ru/news", "https://scientificrussia.ru/news/1224", "https://scientificrussia.ru/news/1223"],
```

Comment:
are these urls enough to collect 100 articles?

Reply:
nope, forgot to change it. done!